#164576233 - Setup Dashboard page

jsamchineme opened the pull request:
###  What this PR does
- Allow users access a dashboard after login
- Implement articles listing on dashboard
- write tests for the dashboard component

### How to test
- Clone this branch
- signup and/or login
- From sidebar on dashboard, click top reads to view article listing


### Screenshots

![image](https://user-images.githubusercontent.com/14825863/54385262-7c8a7c00-4696-11e9-8b07-f0069969230d.png)

### Relevant pivotal tracker stories
https://www.pivotaltracker.com/story/show/164576233

omoefe-dukuye reviewed (commented):
@jsamchineme Great job man. Loving the dark theme. Though it seems the logic in the `componentDidMount()` hook that exists in `Dashboard.jsx` on staging seems to be missing from your work.

ifiokeyo commented on src/components/comments/CommentEntries.jsx:
```diff
@@ -1,8 +1,8 @@
 import './style.scss';
-import LikeNumberGroup from '../../components/LikeNumberGroup';
-import HorizontalLine from '../../components/HorizontalLine';
+import { LikeNumberGroup } from '../../components/like-button';
```
@jsamchineme Javascript adopts the camelcase pattern for filenames, directory names, variables, etc. Please change `lines 4 and 5` to use this pattern.

ifiokeyo commented on src/components/dashboard/Dashboard.jsx:
```diff
@@ -1,14 +1,20 @@
+import { Switch, Route } from 'react-router-dom';
+import { setLoggedInState } from '../../redux/actions/auth';
+import { getOwnProfile } from '../../redux/actions/profile';
+import { TopReads } from '../articles/top-reads';
```
@jsamchineme see above comment.

ifiokeyo commented on src/components/dashboard/Dashboard.jsx:
```diff
@@ -34,7 +40,16 @@ export class Dashboard extends Component {
   render() {
-    return <p>This is the dashboard page placeholder</p>
+    return (
+      <div className={`dashboard-page-wrapper`}>
```
@jsamchineme why is the `className` reference this way ?

jsamchineme committed:
feature(dashboard): Setup Dashboard page

- Allow users access a dashboard after login
- Implement articles listing on dashboard
- write tests for the dashboard component

[Finishes #164576233]
